tests/test_sample_template.py

```diff
@@ -174,3 +164,50 @@ def test_invlaid_sc_call(request):
 
     
 
+
+def test_invlaid_sc_call_exp(request):
+    wallet1 = create_wallet()
+    st_address = request.config.cache.get('st_address', None)
+    request.config.cache.set('wallet1', wallet1)
+    req_json = {
+        "sc_address": st_address,
+        "function_called": "sample_validate_exp",
+        "signers": [
+            WALLET['address']
+        ],
+        "params": {
+            "amount_to_issue": 1,
+            "address_to_issue": wallet1['address']
+        }
+    }
+    response = requests.post(NODE_URL+'/call-sc', json=req_json)
+
+    assert response.status_code == 200
+    unsigned_transaction = response.json()
+    assert unsigned_transaction['transaction']
+    assert len(unsigned_transaction['signatures']) == 0
+    unsigned_transaction['transaction']['fee'] = 1000000
+
+    response = requests.post(NODE_URL+'/sign-transaction', json={
+        "wallet_data": WALLET,
+        "transaction_data": unsigned_transaction
+    })
+
+    assert response.status_code == 200
+    signed_transaction = response.json()
+    assert signed_transaction['transaction']
+    assert signed_transaction['signatures']
+    assert len(signed_transaction['signatures']) == 1
+
+    response = requests.post(
+        NODE_URL+'/validate-transaction', json=signed_transaction)
+    
+    assert response.json()["response"]["valid"] == True
+
+    if TEST_ENV == 'local':
+        response=requests.post(
+            NODE_URL + '/run-updater?add_to_chain_before_consensus=true')
+    else:
+        print('Waiting to mine block')
```

Comment:
check with `get-transaction` API.

Reply:
it is working

Reply:
new assertion is added using get-txn api